ios/brave-ios/Sources/BraveStore/Subscription/SDK/BraveSkusSDK.swift

```diff
@@ -74,6 +75,22 @@ public struct SkusOrder: Codable {
   }
 }
 
+/// An enum representing the Skus SDK Environment
+public enum BraveSkusEnvironment {
+  case beta
+  case nightly
+  case release
+
+  /// Returns the current Skus SDK Environment
+  public static var current: BraveSkusEnvironment {
+    switch AppConstants.buildChannel {
+    case .beta: return .beta
+    case .debug, .nightly: return .nightly
+    case .release: return .release
+    }
+  }
+}
+
```

Comment:
Technically this whole enum is not needed anymore and you could just use the build channel but approving anyways 😅